webextensions/javascript-apis.json

```diff
@@ -4342,6 +4342,9 @@
                   "version_added": false
                 },
                 "firefox": {
+                  "notes": [
+                    "This context is not available if 'ContextType' is accessed through the 'contextMenus' namespace."
```

Comment:
How about the following, so that it is explicit that the type how the type *can* be accessed without reading other MDN pages (my comment also applies to the MDN article):

> Only available at `menus.ContextType`, not via the `contextMenus.ContextType'